docker-compose.yaml

```diff
@@ -3,7 +3,7 @@ services:
   xnat:
     build:
       context: .
-    image: ghcr.io/brown-bnc/xnat:1.8.7.1
+    image: xnat
```

Comment:
Didn't check this the first time around: what does this change do? Is it necessary for adding the DQR plugin?